ext/http/slab.rs

```diff
@@ -77,6 +84,8 @@ macro_rules! http_trace {
   };
 }
 
+pub(crate) use http_trace;
```

Comment:
Was this omitted by mistake?

Reply:
This is actually necessary as we do some tracing elsewhere in the crate now. Macros by default are just module-local. You need to `pub(crate) use` them to make it work elsewhere. It's a terrible syntax, hah.